File: Entities/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace AspCoreVue.Entities
{
    public partial class Address : DefaultEntityProperties
    {
        public Address()
        {
            UserOrder = new HashSet<UserOrder>();
            UserAddress = new HashSet<UserAddress>();
        }

        public int AddressId { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string City { get; set; }
        public string StateCode { get; set; }
        public string ZipCode { get; set; }
        public string CellPhone { get; set; }
        
        public virtual ICollection<UserAddress> UserAddress {get;set;}
        public virtual ICollection<UserOrder> UserOrder { get; set; }
    }
}

File: ClientApp/components/address/Address.ts
export class Address {
    public addressId?: number;
    public addressLine1?: string;
    public addressLine2?: string;
    public city?: string;
    public stateCode?: string;
    public zipCode?: string;

    public Address() {
        this.addressLine1 = '';
        this.addressLine2 = '';
        this.city = '';
        this.stateCode = '';
        this.zipCode = '';
    }
}